docs/book/v1/authorization-server.md

```diff
@@ -6,6 +6,7 @@ for your application.
 Since there are authorization flows that require user interaction,
 **your application is expected to provide the middleware to handle this**.
 
+
```

Comment:
This should be reverted; it adds an extra line between content, which our own linter will reject.